Task 3/script.js

```diff
@@ -11,4 +11,42 @@ Pastaba: Sukurta kortelė, kurioje yra pateikiama vartotojo informacija, turi
 turėti bent minimalų stilių ir būti responsive;
 -------------------------------------------------------------------------- */
 
-const ENDPOINT = 'https://api.github.com/users';
+const ENDPOINT = "https://api.github.com/users";
+
+document.getElementById("btn").addEventListener("click", printUsers);
+
+function printUsers() {
+  let output = document.getElementById("output");
+  output.style.display = "grid";
+  output.style.gridTemplateColumns = "25% 25% 25% 25%";
```

Comment:
Šitą dalį galima iškelti, jei mygtukas bus paspaudžiamas daugiau nei vieną kartą